hw/syn/data/dc.hjson

```diff
@@ -0,0 +1,15 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  tool_srcs: ["{proj_root}/hw/foundry/syn/{tool}/setup.tcl"
+              "{proj_root}/hw/syn/tools/{tool}/start-dc.sh"
+              "{proj_root}/hw/syn/tools/{tool}/run-syn.tcl"
+              "{proj_root}/hw/syn/tools/{tool}/constraints.sdc"]
+
+  // we rely on fusesoc to run lint for us
+  build_cmd:  "export TOP_ENTITY={dut} && mkdir -p REPORTS && ./start-dc.sh "
```

Comment:
exports are supported by the tool. Consider using `"exports: [{key1:value1}, {key2:value2}, ...]"` construct in the hjson.

Reply:
Consider moving the `mkdir` command to `post_compile` (or `compile_result`) as applicable.

Reply:
ok, I see. let me try that...

the mkdir command is needed after running fusesoc, but before starting the tool. I'll move that to another makefile target.